codegen/module.go

```diff
@@ -819,7 +818,7 @@ func readPackageInfo(
 		// uniqueness of the provided package aliases. Note that the default
 		// package is "PackageName".
 		PackageAlias:          defaultAlias + "Static",
-		GeneratedPackageAlias: defaultAlias + "Generated",
+		GeneratedPackageAlias: defaultAlias + "generated",
```

Comment:
looking at the direct above and below, shouldn't we be consistent on all alias cases?

Reply:
only `GeneratedPackageAlias` is used in the header line `package <GeneratedPackageAlias>` according to templates, other alias are used in import package renaming
```golang
import (
    <alias> "path/to/package"
)
```

Reply:
That is correct, my point is since we are doing this, we should keep the package alias style consistent.